k8s/service_controller/model/service_instance.go

```diff
@@ -22,9 +22,9 @@ type LastOperation struct {
 }
 
 type CreateServiceInstanceRequest struct {
```

Comment:
Rather than use SB_ SC_ prefixes, perhaps a better model would be to use golang package for this? That seems cleaner to me. So, perhaps we'll have model. Maybe something like:
model/sc/...
model/sb/...